packages/driver/cypress/e2e/commands/sessions/sessions.cy.js

```diff
@@ -174,7 +174,6 @@ describe('cy.session', { retries: 0 }, () => {
 
         expect(consoleProps).to.deep.eq({
           Command: 'session',
-          Snapshot: 'The snapshot is missing. Displaying current state of the DOM.',
```

Comment:
what happens when a user tries to pin the session command after running in this case? Do we display a session snapshot or does it not even show up because it's a system level command?

Reply:
It just shows the browser at the time the command kicked off